ETC.tsv

```diff
@@ -4618,7 +4618,7 @@ ETC_20150317_004796	Shaman dolls for destruction
 ETC_20150317_004797	Great eyes have whispered you to the right person is a Watono.
 ETC_20150317_004798	Part of the fragmentation barrier is insufficient!
 ETC_20150317_004799	Junior Spirit
-ETC_20150317_004800	Works I was fully collected!
+ETC_20150317_004800	Work was assembled sufficiently!
```

Comment:
Assembled sufficiently implies that the work was put together instead of collected in its entirety. This translation is too liberal. Try "The work was fully collected!"